trillian/ctfe/instance_test.go

```diff
@@ -390,3 +391,58 @@ func TestValidateLogMultiConfig(t *testing.T) {
 		}
 	}
 }
+
+func TestToMultiLogConfig(t *testing.T) {
+	var tests = []struct {
+		desc string
+		cfg  []*configpb.LogConfig
+		want *configpb.LogMultiConfig
+	}{
+		{
+			desc: "one valid log config",
+			cfg: []*configpb.LogConfig{
+				{LogId: 1, Prefix: "test"},
+			},
+			want: &configpb.LogMultiConfig{
+				Backends: &configpb.LogBackendSet{
+					Backend: []*configpb.LogBackend{{Name: "default", BackendSpec: "spec"}},
+				},
+				LogConfigs: &configpb.LogConfigSet{
+					Config: []*configpb.LogConfig{{Prefix: "test", LogId: 1, LogBackendName: "default"}},
+				},
+			},
+		},
+		{
+			desc: "three valid log configs",
+			cfg: []*configpb.LogConfig{
+				{LogId: 1, Prefix: "test1"},
+				{LogId: 2, Prefix: "test2"},
+				{LogId: 3, Prefix: "test3"},
+			},
+			want: &configpb.LogMultiConfig{
+				Backends: &configpb.LogBackendSet{
+					Backend: []*configpb.LogBackend{{Name: "default", BackendSpec: "spec"}},
+				},
+				LogConfigs: &configpb.LogConfigSet{
+					Config: []*configpb.LogConfig{
+						{Prefix: "test1", LogId: 1, LogBackendName: "default"},
+						{Prefix: "test2", LogId: 2, LogBackendName: "default"},
+						{Prefix: "test3", LogId: 3, LogBackendName: "default"},
+					},
+				},
+			},
+		},
+	}
+
+	for _, test := range tests {
+		got, beMap := ToMultiLogConfig(test.cfg, "spec")
+
+		if !proto.Equal(got, test.want) {
+			t.Errorf("TestToMultiLogConfig() got: %v, want: %v (%v)", got, test.want, test.desc)
+		}
+		// Should always produce a 1 element size backend map
+		if got, want := len(beMap), 1; got != want {
+			t.Errorf("TestToMultiLogConfig() backend map size got: %v, want: %v (%v)", got, want, test.desc)
+		}
```

Comment:
Why not feed the multi-backend config to `ValidateLogMultiConfig` here and check it's OK?

Reply:
Now moot because of other changes.